### PR TITLE
ci: run a droplet testnet daily on main

### PR DESCRIPTION
**Pull request by joshuef**
This should run a live WAN testnet, across many droplets whenever we push to the given branch

### CODE REVIEW

File: .github/workflows/daily_droplet_run.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: 'eu-west-2'
+
+jobs:
+  launch-testnet:
+    name: Launch Digital Ocean testnet
+    runs-on: ubuntu-latest
+    steps:
+      - name: Launch testnet
+        uses: joshuef/sn_testnet_action@ImproveAction
```
Comment (lionel1704) [inline .github/workflows/daily_droplet_run.yml:29]: Perhaps we should PR the improvement to maidsafe/sn_testnet_action?

File: .github/workflows/daily_droplet_run.yml [hunk 2]
```diff
@@ -0,0 +1,111 @@
+name: 'sn: Run Client Tests against DO Testnet'
+
+on:
+  schedule:
```
Comment (lionel1704) [inline .github/workflows/daily_droplet_run.yml:4]: All branches?

File: .github/workflows/daily_droplet_run.yml (hunk 1 above)
Comment (joshuef) [reply .github/workflows/daily_droplet_run.yml:29]: ah, that should be in. that needs updated thanks @lionel1704 👍

File: .github/workflows/daily_droplet_run.yml (hunk 2 above)
Comment (joshuef) [reply .github/workflows/daily_droplet_run.yml:4]: ah, i'd understood `schedule` defaulted to the default branch. Lemme double check docs

Comment (joshuef) [reply .github/workflows/daily_droplet_run.yml:4]: Aye, according to this it's whatever is `default`: https://stackoverflow.com/questions/58798886/github-actions-schedule-operation-on-branch

**Review by lionel1704 — APPROVED** [review]

**Pull request merged**